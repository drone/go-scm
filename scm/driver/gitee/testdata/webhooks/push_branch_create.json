{
  "after": "6168d9dae737b47f00c59fafca10c913a6850c3a",
  "before": "0000000000000000000000000000000000000000",
  "commits": null,
  "commits_more_than_ten": true,
  "compare": "https://gitee.com/kit101/drone-yml-test/compare/0000000000000000000000000000000000000000...6168d9dae737b47f00c59fafca10c913a6850c3a",
  "created": true,
  "deleted": false,
  "enterprise": null,
  "head_commit": {
    "added": [
    ],
    "author": {
      "email": "qkssk1711@163.com",
      "id": 1535738,
      "name": "kit101",
      "remark": null,
      "time": "2021-10-08T17:37:47+08:00",
      "url": "https://gitee.com/kit101",
      "user": null,
      "user_name": "kit101",
      "username": "kit101"
    },
    "committer": {
      "email": "qkssk1711@163.com",
      "id": 1535738,
      "name": "kit101",
      "remark": null,
      "time": null,
      "url": "https://gitee.com/kit101",
      "user": null,
      "user_name": "kit101",
      "username": "kit101"
    },
    "distinct": true,
    "id": "6168d9dae737b47f00c59fafca10c913a6850c3a",
    "message": "feat-2\n",
    "modified": [
      "modify.txt"
    ],
    "parent_ids": [
      "00b76e8abd51ae6a96318b3450944b32995f9158"
    ],
    "removed": [
    ],
    "timestamp": "2021-10-08T17:37:47+08:00",
    "tree_id": "59792feaa92f1bc2460eb1fbc0122dd5029ac583",
    "url": "https://gitee.com/kit101/drone-yml-test/commit/6168d9dae737b47f00c59fafca10c913a6850c3a"
  },
  "hook_id": 788005,
  "hook_name": "push_hooks",
  "hook_url": "https://gitee.com/kit101/drone-yml-test/hooks/788005/edit",
  "password": "",
  "project": {
    "clone_url": "https://gitee.com/kit101/drone-yml-test.git",
    "created_at": "2021-03-24T11:24:34+08:00",
    "default_branch": "master",
    "description": "",
    "fork": false,
    "forks_count": 0,
    "full_name": "kit101/drone-yml-test",
    "git_http_url": "https://gitee.com/kit101/drone-yml-test.git",
    "git_ssh_url": "git@gitee.com:kit101/drone-yml-test.git",
    "git_svn_url": "svn://gitee.com/kit101/drone-yml-test",
    "git_url": "git://gitee.com/kit101/drone-yml-test.git",
    "has_issues": true,
    "has_pages": false,
    "has_wiki": true,
    "homepage": "https://gitee.com/kit101/drone-yml-test",
    "html_url": "https://gitee.com/kit101/drone-yml-test",
    "id": 14836026,
    "language": null,
    "license": null,
    "name": "drone-yml-test",
    "name_with_namespace": "kit101/drone-yml-test",
    "namespace": "kit101",
    "open_issues_count": 2,
    "owner": {
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "email": "qkssk1711@163.com",
      "html_url": "https://gitee.com/kit101",
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "remark": null,
      "site_admin": false,
      "type": "User",
      "url": "https://gitee.com/kit101",
      "user_name": "kit101",
      "username": "kit101"
    },
    "path": "drone-yml-test",
    "path_with_namespace": "kit101/drone-yml-test",
    "private": false,
    "pushed_at": "2021-10-08T17:39:43+08:00",
    "ssh_url": "git@gitee.com:kit101/drone-yml-test.git",
    "stargazers_count": 0,
    "svn_url": "svn://gitee.com/kit101/drone-yml-test",
    "updated_at": "2021-10-08T17:39:44+08:00",
    "url": "https://gitee.com/kit101/drone-yml-test",
    "watchers_count": 1
  },
  "push_data": null,
  "pusher": {
    "email": "qkssk1711@163.com",
    "id": 1535738,
    "name": "kit101",
    "url": "https://gitee.com/kit101",
    "user_name": "kit101",
    "username": "kit101"
  },
  "ref": "refs/heads/feat-2-1",
  "repository": {
    "clone_url": "https://gitee.com/kit101/drone-yml-test.git",
    "created_at": "2021-03-24T11:24:34+08:00",
    "default_branch": "master",
    "description": "",
    "fork": false,
    "forks_count": 0,
    "full_name": "kit101/drone-yml-test",
    "git_http_url": "https://gitee.com/kit101/drone-yml-test.git",
    "git_ssh_url": "git@gitee.com:kit101/drone-yml-test.git",
    "git_svn_url": "svn://gitee.com/kit101/drone-yml-test",
    "git_url": "git://gitee.com/kit101/drone-yml-test.git",
    "has_issues": true,
    "has_pages": false,
    "has_wiki": true,
    "homepage": "https://gitee.com/kit101/drone-yml-test",
    "html_url": "https://gitee.com/kit101/drone-yml-test",
    "id": 14836026,
    "language": null,
    "license": null,
    "name": "drone-yml-test",
    "name_with_namespace": "kit101/drone-yml-test",
    "namespace": "kit101",
    "open_issues_count": 2,
    "owner": {
      "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
      "email": "qkssk1711@163.com",
      "html_url": "https://gitee.com/kit101",
      "id": 1535738,
      "login": "kit101",
      "name": "kit101",
      "remark": null,
      "site_admin": false,
      "type": "User",
      "url": "https://gitee.com/kit101",
      "user_name": "kit101",
      "username": "kit101"
    },
    "path": "drone-yml-test",
    "path_with_namespace": "kit101/drone-yml-test",
    "private": false,
    "pushed_at": "2021-10-08T17:39:43+08:00",
    "ssh_url": "git@gitee.com:kit101/drone-yml-test.git",
    "stargazers_count": 0,
    "svn_url": "svn://gitee.com/kit101/drone-yml-test",
    "updated_at": "2021-10-08T17:39:44+08:00",
    "url": "https://gitee.com/kit101/drone-yml-test",
    "watchers_count": 1
  },
  "sender": {
    "avatar_url": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
    "email": "qkssk1711@163.com",
    "html_url": "https://gitee.com/kit101",
    "id": 1535738,
    "login": "kit101",
    "name": "kit101",
    "remark": null,
    "site_admin": false,
    "type": "User",
    "url": "https://gitee.com/kit101",
    "user_name": "kit101",
    "username": "kit101"
  },
  "sign": "5QkJzTgTd5PJ555MAindBUxryzHWESPOMeOz+dIzTFc=",
  "timestamp": "1633686089510",
  "total_commits_count": 13,
  "user": {
    "email": "qkssk1711@163.com",
    "id": 1535738,
    "name": "kit101",
    "url": "https://gitee.com/kit101",
    "user_name": "kit101",
    "username": "kit101"
  },
  "user_id": 1535738,
  "user_name": "kit101"
}