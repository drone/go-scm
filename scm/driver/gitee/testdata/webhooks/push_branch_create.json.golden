{
  "Ref": "refs/heads/feat-2-1",
  "Before": "0000000000000000000000000000000000000000",
  "After": "6168d9dae737b47f00c59fafca10c913a6850c3a",
  "Repo": {
    "ID": "14836026",
    "Namespace": "kit101",
    "Name": "drone-yml-test",
    "Perm": null,
    "Branch": "master",
    "Private": false,
    "Clone": "https://gitee.com/kit101/drone-yml-test.git",
    "CloneSSH": "git@gitee.com:kit101/drone-yml-test.git",
    "Link": "https://gitee.com/kit101/drone-yml-test",
    "Created": "2021-03-24T11:24:34+08:00",
    "Updated": "2021-10-08T17:39:44+08:00"
  },
  "Commit": {
    "Sha": "6168d9dae737b47f00c59fafca10c913a6850c3a",
    "Message": "feat-2\n",
    "Author": {
      "Login": "kit101",
      "Name": "kit101",
      "Email": "qkssk1711@163.com",
      "Date": "2021-10-08T17:37:47+08:00"
    },
    "Committer": {
      "Login": "kit101",
      "Name": "kit101",
      "Email": "qkssk1711@163.com",
      "Date": "2021-10-08T17:37:47+08:00"
    },
    "Link": "https://gitee.com/kit101/drone-yml-test/compare/0000000000000000000000000000000000000000...6168d9dae737b47f00c59fafca10c913a6850c3a"
  },
  "Sender": {
    "Avatar": "https://portrait.gitee.com/uploads/avatars/user/511/1535738_qkssk1711_1578953939.png",
    "Email": "qkssk1711@163.com",
    "Login": "kit101",
    "Name": "kit101"
  }
}